akka-cluster/src/main/resources/reference.conf

```diff
@@ -289,5 +289,10 @@ akka {
       adaptive-group = "akka.cluster.routing.AdaptiveLoadBalancingGroup"
     }
   }
+  
+  coordinated-shutdown-phases {
+    cluster-pre-exiting = []
+    cluster-exiting = [cluster-pre-exiting]
+  }
```

Comment:
yes, that is probably a good idea. then we could add more to each step, e.g. a timeout